doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -33,9 +33,12 @@ Connecting
 The application can use :c:func:`nrf_cloud_connect` to connect to the cloud.
 This API triggers a series of events and actions in the system.
 If the API fails, the application must retry to connect.
-If the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD` Kconfig option is not enabled, the application should monitor the connection socket. :c:func:`nrf_cloud_connect` blocks and returns success when the MQTT connection to the cloud completes.
-If the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD` Kconfig option is enabled, an nRF Cloud library thread monitors the connection socket. :c:func:`nrf_cloud_connect` does not block and returns success if the connection monitoring thread has started.
+If the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD` Kconfig option is not enabled, the application should monitor the connection socket.
+The :c:func:`nrf_cloud_connect` function blocks and returns success when the MQTT connection to the cloud completes.
+If the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD` Kconfig option is enabled, an nRF Cloud library thread monitors the connection socket.
+The :c:func:`nrf_cloud_connect` function does not block and returns success if the connection monitoring thread has started.
```

Comment:
```suggestion
If the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD` Kconfig option is enabled, an nRF Cloud library thread monitors the connection socket.
The :c:func:`nrf_cloud_connect` function does not block and returns success if the connection monitoring thread has started.

```

Reply:
I'm afraid you have not implemented this split quite yet. I meant the second paragraph under the heading "Connecting" would be split into two paragraphs, ending up to two starting with "If the :kconfig:option:...

Reply:
Sorry about that!

Reply:
No worries Georges.